Makefile

```diff
@@ -15,16 +15,16 @@ else
 endif
 	@$(MAKE) $(PKG_TYPE)
 
-rpm:
+RPM:
```

Comment:
interesting.. why does pylint care Makefile targets? dont we limit pylint to `*.py` files?

Reply:
Actually it's not the makefile that pylint cares about. 
https://github.com/rax-maas/ele-agent-meta-packages/blob/80b1a6519c07aca4773bfedafb1e08c7096743f6/Makefile#L5 Here makefile tries to get the pkg_type from the [pkgutils.py](https://github.com/rax-maas/ele-agent-meta-packages/blob/80b1a6519c07aca4773bfedafb1e08c7096743f6/tools/pkgutils.py#L31) and this here was changed from smallcase to uppercase.

Reply:
Due to this makefile tries to run `make DEB` but there was no target with DEB as it was `deb` in makefile.